paper/src/main/java/com/denizenscript/denizen/paper/PaperModule.java

```diff
@@ -15,6 +15,7 @@ public static void init() {
         ScriptEvent.registerScriptEvent(new PlayerEquipsArmorScriptEvent());
         ScriptEvent.registerScriptEvent(new PlayerJumpsPaperScriptEventImpl());
         ScriptEvent.registerScriptEvent(new PlayerSpectatesEntityScriptEvent());
+        ScriptEvent.registerScriptEvent(new EntityKnocksbackEntityScriptEvent());

```

Comment:
This list should be alphabetized (just look at the order of events in your IntelliJ class listing)